lesson13/style.css

```diff
@@ -0,0 +1,121 @@
+body,html{
+    font-family: YuGothic,'Yu Gothic','Yu Gothic UI','ＭＳ ゴシック','ヒラギノ角ゴシック','Hiragino Sans','Hiragino Kaku Gothic ProN','ヒラギノ角ゴ ProN W3', sans-serif;
+    font-size: 16px;
+    color:rgb(70, 70, 70);
+    line-height: 1.2rem;
+    letter-spacing: 0.05rem;
+}
+a{
+    text-decoration: none;
+}
+a:hover{
+    opacity: 0.7;
+    transition: 0.2s;
+}
+.wrapper{
+    height: 100vh;
+}
+
+/*js-list,
+#js-loading*/
+#js-list,#js-loading{
```

Comment:
IMO
I was under the impression that js-prefixed classes were for js files,
so I think it's better to give classes for CSS to those that are handled by CSS.
What do you think?

Reply:
Thank you for your comment!
I certainly think so.
I've styled the element with the js- prefix, so I'm going to add a class for CSS and fix it.

Reply:
I fixed. https://github.com/nakagawamai/js-lesson/pull/15/commits/7e9f836056fb419aed9e12cd828bbb227862dd8f

Reply:
Considering CSS maintainability and level of detail,
it would be better to give classes for CSS to `ul `and `li` as well.

The following code can be used to add classes to li.
`Element.classList.add("hogehoge");`
[MDN](https://developer.mozilla.org/ja/docs/Web/API/Element/classList)

Reply:
Sorry for the lack of understanding.
I added CSS class to ul and li.
https://github.com/nakagawamai/js-lesson/pull/15/commits/c2bf66d57a43c7e0621ea97ec7cc919e9c47e9bd

Reply:
Thank you for your fixed.

Didn't know about the className method!
So it would be an overwrite, not an addition_φ(･_･!
Thank you :)